### PR TITLE
[Azure Stack] Update Compute Version

### PR DESCRIPTION
**Pull request by deathly809**
## Description

Update the compute version.  This is waiting for other PRs to go in, I have already merged from upstream/preview.  Tests have passed locally after importing changes from other PRs.

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by deathly809** [comment]
PS-Sign is here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1203/

File: src/StackAdmin/Compute/AzureRM.Compute.Netcore.psd1 [hunk 1]
```diff
@@ -30,10 +30,10 @@ CompanyName = 'Microsoft Corporation'
 Copyright = 'Microsoft Corporation. All rights reserved.'
 
 # Description of the functionality provided by this module
-Description = 'Microsoft Azure PowerShell - Compute service cmdlets for Azure Resource Manager.  Manages virtual machines, hosted services, and related resources in Azure Resource Manager'
+Description = '[PowerShell .Net Core] Microsoft Azure PowerShell - Compute service cmdlets for Azure Resource Manager.  Manages virtual machines, hosted services, and related resources in Azure Resource Manager'
 
 # Minimum version of the Windows PowerShell engine required by this module
-PowerShellVersion = '3.0'
+PowerShellVersion = '5.1'
```
Comment (bganapa) [inline src/StackAdmin/Compute/AzureRM.Compute.Netcore.psd1]: lets remove *netcore* files

**Comment by maddieclayton** [comment]
Couple of comments:

Please delete netcore psd1 and csproj files
PowerShell version in psd1 should be 5.0 not 3.0
We've been removing the module specific sln files, as we want the entire StackAdmin sln to be able to build them all - please remove the Compute.sln
azure-powershell/src/StackAdmin/Compute/VhdManagement/Properties/AssemblyInfo.cs - version should match
azure-powershell/src/StackAdmin/Compute/Sync/Properties/AssemblyInfo.cs - version should match
Where is the csproj for Sync?

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/help/Set-AzureRmStorageAccount.md [hunk 2]
```diff
@@ -1,472 +0,0 @@
-﻿---
-external help file: Microsoft.Azure.Commands.Management.Storage.dll-Help.xml
-ms.assetid: 4D7EEDD7-89D4-4B1E-A9A1-B301E759CE72
-online version: 
-schema: 2.0.0
----
-
```
Comment (bganapa) [inline src/StackAdmin/Storage.Management/Commands.Management.Storage/help/Set-AzureRmStorageAccount.md]: Lets add the storage mds back

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/Microsoft.Azure.Commands.Management.Storage.dll-Help.psd1 [hunk 3]
```diff
@@ -0,0 +1,111 @@
+#
+# Module manifest for module 'Microsoft.Azure.Commands.Management.Storage'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 04/01/2015
+#
+
+@{
+
```
Comment (bganapa) [inline src/StackAdmin/Storage.Management/Commands.Management.Storage/Microsoft.Azure.Commands.Management.Storage.dll-Help.psd1]: this one got deleted in preview, you are adding it back

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/help/Set-AzureRmStorageAccount.md (hunk 2 above)
Comment (deathly809) [reply src/StackAdmin/Storage.Management/Commands.Management.Storage/help/Set-AzureRmStorageAccount.md]: Another PR.

File: src/StackAdmin/Storage.Management/AzureRM.Storage.psd1 [hunk 4]
```diff
@@ -0,0 +1,132 @@
+﻿#
+# Module manifest for module 'PSGet_AzureRM.Storage'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 7/6/2017
+#
+
```
Comment (bganapa) [inline src/StackAdmin/Storage.Management/AzureRM.Storage.psd1]: repls remove this file

File: src/StackAdmin/Resources/Commands.Resources/AzureRM.Resources.Netcore.psd1 [hunk 5]
```diff
@@ -0,0 +1,173 @@
+﻿#
+# Module manifest for module 'PSGet_AzureRM.Resources'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 4/3/2017
+#
+
+@{
```
Comment (bganapa) [inline src/StackAdmin/Resources/Commands.Resources/AzureRM.Resources.Netcore.psd1]: this one was removed in preview

File: src/StackAdmin/Resources/Commands.ResourceManager/Cmdlets/Commands.Resources.Rest.Netcore.csproj [hunk 6]
```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\..\..\..\tools\Common.Netcore.Dependencies.targets" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>Microsoft.Azure.Commands.ResourceManager.Cmdlets</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Commands.ResourceManager.Cmdlets</RootNamespace>
```
Comment (bganapa) [inline src/StackAdmin/Resources/Commands.ResourceManager/Cmdlets/Commands.Resources.Rest.Netcore.csproj]: this one does not exist in preview

File: src/StackAdmin/Network/Commands.Network/AzureRM.Network.Netcore.psd1 [hunk 7]
```diff
@@ -0,0 +1,365 @@
+﻿#
+# Module manifest for module 'PSGet_AzureRM.Network'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 4/3/2017
+#
+
```
Comment (bganapa) [inline src/StackAdmin/Network/Commands.Network/AzureRM.Network.Netcore.psd1]: pls remove this file

File: src/StackAdmin/Network/Commands.Network.Test/packages.config [hunk 8]
```diff
@@ -7,4 +7,6 @@
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Storage" version="6.1.0-preview" targetFramework="net45" />
-</packages>
\ No newline at end of file
+  <package id="Microsoft.Azure.Test.Framework" version="1.0.6179.26854-prerelease" targetFramework="net45" />
```
Comment (bganapa) [inline src/StackAdmin/Network/Commands.Network.Test/packages.config]: pls remove the chnages in this file

File: src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 9]
```diff
@@ -64,6 +64,66 @@
...
+    </Reference>
+    <Reference Include="Microsoft.Threading.Tasks.Extensions, Version=1.0.12.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.Bcl.Async.1.0.168\lib\net40\Microsoft.Threading.Tasks.Extensions.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Threading.Tasks.Extensions.Desktop, Version=1.0.168.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.Bcl.Async.1.0.168\lib\net40\Microsoft.Threading.Tasks.Extensions.Desktop.dll</HintPath>
+    </Reference>
+    <Reference Include="Moq, Version=4.2.1510.2205, Culture=neutral, PublicKeyToken=69f491c39445e920, processorArchitecture=MSIL">
```
Comment (bganapa) [inline src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj]: this file should not be touched by your PR

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/Microsoft.Azure.Commands.Management.Storage.dll-Help.psd1 (hunk 3 above)
Comment (deathly809) [reply src/StackAdmin/Storage.Management/Commands.Management.Storage/Microsoft.Azure.Commands.Management.Storage.dll-Help.psd1]: Done.

File: src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj (hunk 9 above)
Comment (deathly809) [reply src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj]: Hmmm I took all incoming changes from merge.

**Comment by bganapa** [comment]
ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1223/

**Comment by bganapa** [comment]
psnetsign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/179/

**Comment by deathly809** [comment]
@maddieclayton Everything look good now?

**Pull request merged**